Olympus.Stratis/initPlayerLocal.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: Theseus Incorporated
+ *
+ * initPlayerLocal.sqf
+ * Initialization of scripts local to player
+ * Executed locally when player joins mission (includes both mission start and JIP)
+ * Executed after initServer and before initPlayerServer and init.
+ *
+ * Arguments: [player:Object, didJIP:Boolean]
+ *
+ */
```

Comment:
This header should be changed to ACE standard.
Should be standardized with template as well.